packages/mobile/src/components/CodeRow.test.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import 'react-native'
+import { render } from 'react-native-testing-library'
+import CodeRow, { CodeRowStatus } from 'src/components/CodeRow'
+
+describe('CodeRow', () => {
+  it('renders correctly for all CodeRowStatus states', () => {
+    ;[
+      CodeRowStatus.DISABLED,
+      CodeRowStatus.INPUTTING,
+      CodeRowStatus.PROCESSING,
+      CodeRowStatus.RECEIVED,
+      CodeRowStatus.ACCEPTED,
+    ].map((status) => {
+      const { toJSON } = render(
+        <CodeRow
+          status={status}
+          inputValue={'test'}
+          inputPlaceholder={'placeholder'}
+          onInputChange={jest.fn()}
+          shouldShowClipboard={jest.fn()}
+        />
+      )
+      expect(toJSON()).toMatchSnapshot()
```

Comment:
Nothing to act on right now, I think I've already mentioned that in the past but I'm not exactly convinced about the value of these snapshots tests.
It's good that it renders but it might as well render something completely unusable and still pass the test.

Reply:
I somewhat agree but often find myself short of time and the more in depth tests take longer to write.